src/components/app/Root.jsx

```diff
@@ -20,58 +20,65 @@ class Root extends React.Component {
     };
 
     render() {
-        const { d2 } = this.props;
+        const { isAdmin } = this.props;
```

Comment:
`isAdmin` missing from propTypes

Reply:
Yeah I originally had it set in ```state``` everywhere and moved to props in the last commit. Will add to propTypes in the next commit.